### PR TITLE
Added a variable + slice function to control how many AZs will be deployed

### PR DESCRIPTION
**Pull request by ivan-pinatti**
Hi @aknysh,

This is the PR to control how many AZs it will be deployed, I tested with the new VPC example and it worked like a charm.

Cheers,

Ivan Pinatti

### CODE REVIEW

File: examples/existing_vpc_existing_subnets/main.tf [hunk 1]
```diff
@@ -2,6 +2,10 @@ provider "aws" {
   region = "us-west-2"
 }
 
+variable "azs_to_deploy" {
```
Comment (osterman) [inline examples/existing_vpc_existing_subnets/main.tf]: Let's rename this to `max_availability_zones` as it's more clear that it's a limit.

**Comment by ivan-pinatti** [comment]
@osterman renaming done. :)

**Comment by osterman** [comment]
@ivan-pinatti, thanks! I'll defer to @aknysh for final approval since he's most invested in this! =)

**Comment by aknysh** [comment]
@ivan-pinatti  looks good.
Can you add the same changes to `README`?
It has the same code as the examples.
Thanks

**Comment by ivan-pinatti** [comment]
@aknysh done.

**Comment by aknysh** [comment]
@ivan-pinatti  approved and merged
https://github.com/cloudposse/terraform-aws-jenkins/releases/tag/0.3.2

Thanks for your contribution

**Pull request merged**